libs/vkd3d/swapchain.c

```diff
@@ -1360,7 +1377,15 @@ static HRESULT d3d12_swapchain_create_vulkan_swapchain(struct d3d12_swapchain *s
     height = max(height, surface_caps.minImageExtent.height);
     height = min(height, surface_caps.maxImageExtent.height);
 
-    if (width != swapchain->desc.Width || height != swapchain->desc.Height)
+    if (surface_caps.maxImageExtent.width == 0 || surface_caps.maxImageExtent.height == 0)
+    {
+        /* This is expected behavior on Windows when a HWND minimizes.
+         * We must retry creating a proper swapchain later.
+         * Since we can always fall back to user buffers,
+         * we can pretend that we have an active swapchain. */
+        WARN("Window is minimized, cannot create swapchain at this time.\n");
```

Comment:
This is probably going to spam like hell if anyone minimises anything... TRACE?

Reply:
Actually should be okay, but as a warning it might be confusing...

Reply:
Fixed.